src/App.jsx

```diff
@@ -1,16 +1,17 @@
+
 import React from "react";
-import Header from "./components/Header";
-import Nav from "./components/Nav";
-import ToDoContainer from "./components/ToDoContainer";
+import { TaskProvider } from "./context/Task/TaskProvider";
+import { UiProvider } from "./context/ui/UiProvider";
```

Comment:
UI stands for User Interface, I think it's not descriptive enough IMHO. I'd change the name to PopUpProvider or ModalProvider. WDYT?

Reply:
That's good of a review @aleksanderwalczuk .
Would you mind becoming a maintainer of the project ?

Reply:
Sure thing, thanks @SauveJeanLuc!

Reply:
@all-contributors add @aleksanderwalczuk for Maintenance

Reply:
Check invite @aleksanderwalczuk

Reply:
Got it.